src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/catalog_price_rule.yml

```
@@ -54,3 +54,13 @@ admin_catalog_price_rules_bulk_delete:
     _controller: 'PrestaShopBundle\Controller\Admin\Sell\Catalog\CatalogPriceRuleController::bulkDeleteAction'
     _legacy_controller: AdminSpecificPriceRule
 #  _legacy_link: AdminSpecificPriceRule:submitBulkdeletespecific_price_rule
+
+admin_catalog_price_rules_product_form_list:
```

```suggestion
admin_catalog_price_rules_list:
```
As I said in the controller I don't think we must explicitly link this API to the product form